mintapi/signIn.py

```diff
@@ -118,13 +120,6 @@ def get_email_code(imap_account, imap_password, imap_server, imap_folder, delete
             subject = str(x)
             logger.debug("DEBUG: SUBJECT:", subject)
 
-            x = email.header.make_header(email.header.decode_header(msg["From"]))
-            frm = str(x)
-            logger.debug("DEBUG: FROM:", frm)
-
-            if not re.search("do_not_reply@intuit.com", frm, re.IGNORECASE):
```

Comment:
@tsdg112 is there some sort of a domain we could check on the email?

Reply:
I think that we already have sufficient email validation after removing the from check.  If you still want to retain this check, the from field needs to be a configurable option, since it differs for each duck.com email address.

Reply:
@tsdg112 I think this should be a separate PR isolated to a configurable option with the default as the email now.

Reply:
Thats fine with me.  Should I restore the deleted code for this PR?
\

Reply:
@tsdg112 yes please! Thanks!

Reply:
@tsdg112 can you restore the subject back to its original spot so that it doesn't show up as a diff in the PR?